cEP-0009.md

```diff
@@ -0,0 +1,181 @@
+| Metadata     |                                                          |
+| ------------ |----------------------------------------------------------|
+| cEP          | 9                                                        |
+| Version      | 0.1                                                      |
+| Title        | Extract and utilize information from project files       |
+| Authors      | Satwik Kansal <satwikkansal@gmail.com>                   |
+| Status       | Proposed                                                 |
+| Type         | Feature                                                  |
+
+## Abstract
+
+There are many kind of project files like ".editorconfig", "Gruntfile.js",
+etc. that contain meta-information that can used by tools like
```

Comment:
There's no maximum line length. Use long lines for paragraphs. Otherwise markdown will look weird sometimes.